crypto/build.rs

```diff
@@ -0,0 +1,17 @@
+extern crate gcc;
+
+use std::env;
+use std::path::Path;
+
+fn main() {
+    gcc::Config::new()
+        .compiler(&Path::new("arm-none-eabi-gcc"))
+        .flag("-mcpu=cortex-m4")
```

Comment:
This library is suppose to work with m4 (but the files are compatible with m0 as they were originally written for m0). Should I change/rename?